lib/fog/ovirt/requests/compute/v3/list_quotas.rb

```diff
@@ -11,7 +11,7 @@ class Mock
           def list_quotas(_filters = {})
             xml = read_xml "quotas.xml"
             Nokogiri::XML(xml).xpath("/quotas/quota").map do |q|
-              ovirt_attrs OVIRT::Quotas.new(self, q)
```

Comment:
OVIRT::Quotas doesn't exist, I noticed this when writing tests for quotas in foreman and trying to stub the list, it produced an error on `uninitialized constant OVIRT::Quotas`